domains/nidun.json

```diff
@@ -0,0 +1,11 @@
+{
+    "description": "new update to request new cname",
+    "repo": "https://github.com/nldun/ininidun.github.io",
+    "owner": {
+        "username": "nldun",
+        "email": "rezkyto@gmail.com",
```

Comment:
yea you forgot to remove the comma at line 6, please remove that

Reply:
okay

Reply:
done i remove the cOMMAAAA sirr